src/components/models.js

```diff
@@ -1,20 +1,20 @@
 import React, { Component } from 'react';
 import { render } from 'react-dom';
-import SliderContent from './sliderContent';
+import Carousel from './carousel';
+import $ from 'jquery';
```

Comment:
why is "import $ from 'jquery'" necessary?

Reply:
It's not, good catch!